JMS/activemq/docker-compose.yml

```diff
@@ -6,4 +6,4 @@ services:
       - "8000:8000"
     hostname: "kaazing.example.com"
   activemq:
-    build: broker
+    image: webcenter/activemq:latest
```

Comment:
@AdrianCozma If I replace the latest version with 5.10.0, I get the following error at gateway start:
```
gateway_1   | INFO  Started server successfully in 1.516 secs at 2016-08-18 11:37:02
gateway_1   | Exception in thread "ActiveMQ Connection Executor: tcp://activemq/172.18.0.2:61616@43328" java.lang.NullPointerException
gateway_1   |   at org.apache.activemq.ActiveMQConnection$5.run(ActiveMQConnection.java:2005)
gateway_1   |   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
gateway_1   |   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
gateway_1   |   at java.lang.Thread.run(Thread.java:745)
```